src/style.css

```diff
@@ -206,72 +234,99 @@ h5 {
 }
 
 .contact-button {
-  padding: 1vh 3vh;
+  padding: 1vw 3vw;
   background-color: white;
   border: 2px solid tomato;
   border-radius: 4px;
-  font-size: 2vh;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.6vw;
 }
 
 .btn-comment {
-  padding: 1vh 3vh;
+  padding: 1vw 3vw;
   background-color: white;
   border: 2px solid tomato;
   border-radius: 4px;
-  font-size: 2vh;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.6vw;
+  color: grey;
 }
 
-.contact-button:hover,
-.btn-comment:hover {
+.contact-button:hover {
   background-color: tomato;
   border: 3px solid darkgrey;
   color: white;
   border-radius: 4px;
   cursor: pointer;
 }
 
-/* @media only screen and (min-width: 768px) {
-  .work-item-popup {
-    margin: 77px 16vw;
-  }
+.btn-comment:hover {
+  background-color: tomato;
+  border: 3px solid white;
+  color: white;
+  border-radius: 4px;
+  cursor: pointer;
 }
 
-@media only screen and (min-width: 1440px) {
-  .work-item-popup {
-    margin: 77px 22vw;
-  }
-} */
-
 /* Item details */
 
 .popup-section {
-  width: 100%;
-  height: 100%;
   position: fixed;
+  z-index: 100;
   top: 0;
   left: 0;
-  backdrop-filter: blur(4px);
+  width: 100%;
+  height: 100vh;
+  opacity: 95%;
+  backdrop-filter: blur(80px);
+  transition: height 1s ease;
+  padding: 20px;
+}
+
+@media only screen and (min-width: 992px) {
+  .popup-section {
+    padding: 50px 100px;
+  }
+}
+
+@media only screen and (min-width: 1200px) {
+  .popup-section {
+    padding: 100px 200px;
+  }
 }
 
 .popup {
-  width: calc(100vw - 40%);
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  border: 5px solid grey;
+  border-radius: 10px;
+  overflow: auto;
+  display: flex;
+  flex-direction: column;
+  padding: 10px 40px;
+}
+
+/*
+.popup {
+  width: calc(100vw - 20%);
```

Comment:
Nice working on the design but I believe you should remove that commented code from your file because it is not a good coding practice to leave the commented code in your project. 😄